source/CommandLine/main.cpp

```diff
@@ -115,7 +115,7 @@ int VIREO_MAIN(int argc, const char * argv[])
 void Vireo::RunExec() {
     TypeManagerRef tm = gShells._pUserShell;
     TypeManagerScope scope(tm);
-    Int32 state = tm->TheExecutionContext()->ExecuteSlices(400, 10000000);
+    Int32 state = tm->TheExecutionContext()->ExecuteSlices(10000, 4);
```

Comment:
Please add a comment as to why we decided to change the value. I know is not clear what would be the best value, but having a history of changes with explanations might help in the future.

Reply:
Done; added a reference to the other place in JS with a note they should be in sync.